CMakeLists.txt

```diff
@@ -55,7 +53,7 @@ if (HAVE_QHULL_2011)
 endif()
 
 include_directories(include)
-include_directories(SYSTEM ${EIGEN_INCLUDE_DIRS} ${Boost_INCLUDE_DIR} ${ASSIMP_INCLUDE_DIRS} ${OCTOMAP_INCLUDE_DIRS})
+include_directories(SYSTEM ${EIGEN3_INCLUDE_DIRS} ${Boost_INCLUDE_DIR} ${ASSIMP_INCLUDE_DIRS} ${OCTOMAP_INCLUDE_DIRS})
```

Comment:
Please use `EIGEN3_INCLUDE_DIR` (without the trailing `S`) here.
`DIRS` is only around with Eigen3.3 which is officially still beta. (debian packaging stuff doesn't mean it's already released...)
Eigen3.2 provides a FindEigen3.cmake that provides only `EIGEN3_INCLUDE_DIR` and that one is also available in 3.3.